CHANGELOG.md

```diff
@@ -178,6 +179,7 @@ Ref: https://keepachangelog.com/en/1.0.0/
 * [\#10897](https://github.com/cosmos/cosmos-sdk/pull/10897) Fix: set a non-zero value on gas overflow.
 * [#9790](https://github.com/cosmos/cosmos-sdk/pull/10687) Fix behavior of `DecCoins.MulDecTruncate`.
 * (crypto) [#11027] Remove dependency on Tendermint core for xsalsa20symmetric.
+* (x/authz) [\#10447](https://github.com/cosmos/cosmos-sdk/pull/10447) Fix authz `NewGrant` expiration check.
```

Comment:
This isn't accurate anymore. Also, the entry should probably be under `### State Machine Breaking`.